java/ql/src/Security/CWE/CWE-113/NettyResponseSplitting.ql

```diff
@@ -7,35 +7,82 @@
  * @problem.severity error
```

Comment:
Would you like to see the name of this file changed, or is this fine?

Reply:
I would say it's fine, the CWE is called `HTTP Response Splitting` too, even if the injection can technically happen in a request. I see you added CAPEC-105 precisely for this reason — see my other comment.